### PR TITLE
Feature/user guide link

### PR DESCRIPTION
**Pull request by esotrope**

### CODE REVIEW

File: webapp/src/main/webapp/src/app/app.component.ts [hunk 1]
```diff
@@ -38,25 +35,19 @@ export class AppComponent {
...
   }
 
   ngOnInit() {
 
     this.userService.getCurrentUser().subscribe(user => {
       if (!isNullOrUndefined(user)) {
-        this._user = {
-          firstName: user.firstName,
-          lastName: user.lastName
-        };
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/app.component.ts]: is there a reason why we were only passing a 'view' of the user to the ui? I changed it to pass the whole user object.

**Comment by jtreuting** [comment]
Isn't there more than 1 place for the user guide link.  Top nav and homepage right?

**Comment by esotrope** [comment]
@jtreuting added href to link on homepage

File: webapp/src/main/webapp/src/app/home/home.component.html [hunk 2]
```diff
@@ -42,7 +42,7 @@ <h1 class="blue-dark h1 mb-md">{{'labels.homepage.title' | translate}}</h1>
       <div class="welcome-container well">
         <p class="lead">{{'labels.homepage.welcome.title' | translate}}</p>
         <p>{{'labels.homepage.welcome.sub-title' | translate}}</p>
-        <p><a href="javascript:void(0)" (click)="showComingSoon('User Guide')" class="btn btn-primary">{{'labels.navigation.user-guide' | translate}}</a></p>
+        <p><a href="{{user.configuration.userGuideUrl}}" target="_blank" class="btn btn-primary">{{'labels.navigation.user-guide' | translate}}</a></p>
```
Comment (jtreuting) [inline webapp/src/main/webapp/src/app/home/home.component.html]: i think we need to remove the showComingSoon() method from `home.component.ts`

File: webapp/src/main/webapp/src/app/app.component.ts (hunk 1 above)
Comment (bgsavage) [reply webapp/src/main/webapp/src/app/app.component.ts]: Just all that was needed at the time.

File: webapp/src/main/webapp/src/app/home/home.component.html (hunk 2 above)
Comment (esotrope) [reply webapp/src/main/webapp/src/app/home/home.component.html]: removed with latest commit

File: webapp/src/main/webapp/src/app/app.component.ts (hunk 1 above)
Comment (esotrope) [reply webapp/src/main/webapp/src/app/app.component.ts]: 👍

**Pull request merged**